akka-docs/rst/images/persistence-drop-event-serializer.svg

```diff
@@ -0,0 +1,3 @@
71.03075 43.943867 C 485.93475 43.943867 498.03075 40.364439 498.03075 35.954071" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><text transform="translate(449.03075 65.908152)" fill="black"><tspan font-family="Helvetica" font-size="10" font-weight="500" x="5.6010742" y="10" textLength="32.797852">Journal</tspan></text></g><line x1="444.03074" y1="82.74999" x2="369.26871" y2="82.74999" marker-end="url(#FilledArrow_Marker)" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><rect x="394.7397" y="70.74999" width="19" height="24" fill="white"/><text transform="translate(399.7397 75.74999)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".49804688" y="11" textLength="8.0039062">E</tspan></text><line x1="444.03074" y1="60.612232" x2="369.26871" y2="60.612232" marker-end="url(#FilledArrow_Marker)" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><rect x="394.2397" y="48.612232" width="20" height="24" fill="white"/><text transform="translate(399.2397 53.612232)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".33300781" y="11" textLength="9.3339844">O</tspan></text><line x1="187.09439" y1="84.75" x2="123.08368" y2="84.75" marker-end="url(#FilledArrow_Marker)" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><rect x="145.63266" y="72.75" width="19" height="24" fill="white"/><text transform="translate(150.63266 77.75)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".49804688" y="11" textLength="8.0039062">E</tspan></text><g id="id57_Graphic"><rect x="296.34997" y="27.964282" width="63.018734" height="79.897957" fill="white"/><rect x="296.34997" y="27.964282" width="63.018734" height="79.897957" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><text transform="translate(301.34997 60.91326)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x="1.5015545" y="11" textLength="50.015625">Serializer</tspan></text></g><line x1="296.98822" y1="82.87499" x2="254.58112" y2="82.87499" marker-end="url(#FilledArrow_Marker)" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><rect x="262.90395" y="70.87499" width="19" height="24" fill="white"/><text transform="translate(267.90395 75.87499)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".49804688" y="11" textLength="8.0039062">E</tspan></text><line x1="296.98822" y1="60.11223" x2="260.18112" y2="60.11223" marker-end="url(#NonNavigable_Marker)" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="1"/><rect x="263.40395" y="48.11223" width="18" height="24" fill="white"/><text transform="translate(268.40395 53.11223)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".33496094" y="11" textLength="7.330078">T</tspan></text></g></g></svg>
```

Comment:
Hm, here I mean that "O is old, when it gets out of the journal we reject it; when E gets out we pass it through"